docker-compose.yml

```diff
@@ -5,8 +5,8 @@ services:
     build: .
     command: >
       bash -c "python setup.py develop &&  \
-               mkdir -p models/styletts2  && \
-               aws s3 sync s3://uberduck-models-us-west-2/prototype/styletts2 models/styletts2 && \ 
```

Comment:
it looks like this branch is a bit out of date with main, can you run:
```
git checkout more-tts-archs
git pull --rebase origin main
<resolve any merge conflicts>
git push -f
```

Reply:
aye aye captain